cmake/NSIS.template.in

```diff
@@ -555,6 +555,14 @@ Section /o "Open OBJ files with Cura"
     WriteRegStr HKCR "Cura OBJ model file\shell\open\command" "" '"$INSTDIR\Cura.exe" "%1"'
 SectionEnd
 
+Section /o "Open 3MF files with Cura"
+    WriteRegStr HKCR .3mf "" ""
```

Comment:
"3D Manufacturing Format"

I'll just merge this, add the string and component test it tomorrow morning.